apps/src/Sound.js

```diff
@@ -339,22 +340,55 @@ Sound.prototype.getPlayableFile = function() {
   return false;
 };
 
+Sound.prototype.getPlayableBytes = function() {
+  try {
+    if (!window.Audio) {
+      return false;
+    }
+
+    var audioTest = new window.Audio();
+    if (
+      this.config.hasOwnProperty('bytes') &&
+      audioTest.canPlayType('audio/wav')
+    ) {
+      return this.config.bytes;
+    }
+  } catch (e) {}
```

Comment:
Blanket catch statements are often hard to debug. Consider doing a console.warn here or some other sort of logging. (Also, I don't have an understanding of this use case, so if this makes more sense go ahead and keep it the way it is)